sklearn/neighbors/approximate.py

```diff
@@ -93,7 +93,7 @@ class GaussianRandomProjectionHash(ProjectionToHashMixin,
                                    GaussianRandomProjection):
     """Use GaussianRandomProjection to produce a cosine LSH fingerprint"""
     def __init__(self,
-                 n_components=8,
+                 n_components=32,
```

Comment:
is the error captured if not 32? should it be a parameter if only one  param value works?